examples/slc/S1A_IW_SL1__1_SH_20141031T095929_20141031T100002_003072_003842/S1A_IW_SL1__1_SH_20141031T095929_20141031T100002_003072_003842.json

```diff
@@ -0,0 +1,278 @@
+{
+  "type": "Feature",
+  "stac_version": "1.0.0",
+  "id": "S1A_IW_SL1__1_SH_20141031T095929_20141031T100002_003072_003842",
+  "properties": {
+    "sar:frequency_band": "C",
+    "sar:center_frequency": 5.405,
+    "sar:observation_direction": "right",
+    "sar:instrument_mode": "IW",
+    "sar:polarizations": [
+      "HH"
+    ],
+    "sar:product_type": "SLC",
+    "sat:platform_international_designator": "0000-000A",
+    "sat:orbit_state": "descending",
+    "sat:absolute_orbit": 3072,
+    "sat:relative_orbit": 25,
+    "sat:anx_datetime": "2014-10-31T09:29:49.126914Z",
+    "processing:facility": "Sentinel-1 IPF",
+    "processing:level": "L1",
+    "processing:software": {
+      "Sentinel-1 IPF": "002.36"
+    },
+    "proj:epsg": 4326,
+    "proj:bbox": [
+      -53.661625,
+      69.582458,
+      -45.827332,
+      71.715393
+    ],
+    "proj:shape": [
+      21032,
+      14733
+    ],
+    "proj:transform": [
+      0.0005317513744654858,
+      0.0,
+      -53.661625,
+      0.0,
+      -0.00010141379802206178,
+      71.715393
+    ],
+    "proj:centroid": {
+      "lat": 70.65156,
+      "lon": -49.86285
+    },
+    "providers": [
+      {
+        "name": "ESA",
+        "roles": [
+          "producer",
+          "processor",
+          "licensor"
+        ],
+        "url": "https://earth.esa.int/eogateway"
+      }
+    ],
+    "platform": "sentinel-1a",
+    "constellation": "sentinel-1",
+    "start_datetime": "2014-10-31T09:59:31.293840Z",
+    "end_datetime": "2014-10-31T09:59:58.717016Z",
+    "s1:instrument_configuration_ID": "3",
+    "s1:datatake_id": "14402",
+    "s1:product_timeliness": "Fast-24h",
+    "s1:orbit_source": "RESORB",
+    "s1:slice_number": "2",
+    "s1:total_slices": "10",
+    "s1:swaths": [
+      "IW1",
+      "IW2",
+      "IW3"
+    ],
+    "s1:shape": [
+      21032,
+      14733
+    ],
```

Comment:
This is a duplicate of `proj:shape`, so should be removed IMO?

Reply:
This is due to the fact I copied the initial version from GRD. Shall I also fix it there, or do this in a separate PR?

Reply:
Let's do it in a separate PR, thanks.